site/content/features/typepatterns.md

```diff
@@ -1,12 +1,11 @@
 ---
 title: Pattern Matching for switch (JEP 427)
```

Comment:
The title should probably now also refer to JEP 441.

Reply:
Fixing right now